### DISCUSSION TITLE
GRID大作业第三小问依赖文件问题

### DISCUSSION THREAD

**Discussion by MHammerJung** [physics-data/faq#150] [Q&A] [answered]
在GRID大作业的第三小问calculate_response中，提到需要使用第二问生成的light_curve.h5文件，但该文件的能量区间为0-300keV，但EBOUND文件中输出能谱的能量区间却是0-2000keV，所以似乎无法用第二问生成的文件做出输出能谱。但是Makefile中的依赖项确实是light_curve.h5，请问这种情况该如何处理呢？

**Comment by heroxbd** [comment]
@babychen9816

**Comment by quxnb** [comment, marked as answer]
light_curve.h5 文件包括所有的能量区间, 只有绘图的时候绘制出相应能量区间的光变曲线

**Reply by MHammerJung** [reply]
谢谢助教，明白题意了
